commit 37fee735b8c14ba6e97763a726ada1b4ff699776
Author: Ian Campbell <ian.campbell@citrix.com>
Date:   Wed Sep 30 11:57:20 2009 +0100

    xenserver: Use first-class pool vswitch controller configuration field
    
    This has been implemented in our vswitch branch to trunk so use it
    instead of the other-config key.

diff --git a/xenserver/etc_xapi.d_plugins_vswitch-cfg-update b/xenserver/etc_xapi.d_plugins_vswitch-cfg-update
index 7472d1b..4db8cc3 100755
--- a/xenserver/etc_xapi.d_plugins_vswitch-cfg-update
+++ b/xenserver/etc_xapi.d_plugins_vswitch-cfg-update
@@ -47,7 +47,7 @@ def update(session, args):
         raise XenAPIPlugin.Failure("MORE_THAN_ONE_POOL_FOR_HOST", [])
     pool = session.xenapi.pool.get_record(pools[0])
     try:
-        controller = pool["other_config"]["vSwitchController"]
+        controller = pool["vswitch_controller"]
     except KeyError, e:
         controller = ""
     currentController = vswitchCurrentController()
